frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccess.spec.tsx

```diff
@@ -25,6 +26,7 @@ describe('Project Access', () => {
         loaded: false,
         loadError: {},
       },
+      roles: defaultAccessRoles,
```

Comment:
Again. Because my recommendation is to change the hook return value, you need to change this test as well:

```suggestion
      roles: {
        data: defaultAccessRoles,
        loaded: true,
      },
```